tools/mslite-script.py

```diff
@@ -0,0 +1,23 @@
+
+def main():
```

Comment:
See below. This should not be needed.

Reply:
We need it to process mslite-schema.js generated by flatc.js to fit the Netron framework. Currently, MindSpore Lite is used in many models and scenarios without bugs. However, if we modify its model.fbs and ops.fbs , it will have a great impact. Can we keep the file mslite-script.py?

Reply:
`op.primitive.value.constructor.name` should return the same result as `op.primitive.type`.